package.json

```diff
@@ -1,7 +1,7 @@
 {
     "name": "@guardian/discussion-rendering",
     "description": "This codebase started as a hack day project by @gtrufitt and @nicl. The purpose is parity of the existing discussion application on Frontend using the discussion API (search for Private Repo).",
-    "version": "2.2.6",
```

Comment:
Isn't this on master?

Reply:
Keen eyes! I forgot to merge the other PR. Done now and merged into here